editor/plugins/org.fusesource.ide.wsdl2rest/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,49 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Fuse wsdl2rest Wrapper Plugin
+Bundle-SymbolicName: org.fusesource.ide.wsdl2rest
+Bundle-Version: 10.3.0.qualifier
```

Comment:
as version has been upgraded will need to modify to 11.0.0 (and rebase on master)